File: Ecoach Android/Ecoach Android/app/src/main/java/com/example/testapi/Matchs.java
package com.example.testapi;

import java.util.Date;

public class Matchs {

    private String id;
    private String numero_match;
    private Date date_heure_match;
    private String categorie_equipe_local;
    private String score_match;
    private String equipe_local;
    private String equipe_adverse;
    private String lieu_match;
    private String lien_feuille_de_match;

    @Override
    public String toString() {
        return "Numero de match : " + this.numero_match + " | Date et Heure du match : " + this.date_heure_match + " | Equipe local : " + this.equipe_local + " | Equipe adverse : " + this.equipe_adverse + " | Lieu du match : " + this.lieu_match;
    }

}

File: c#/c#/DemoEF/Data/Models/Matchs.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

#nullable disable

namespace DemoEF.Data.Models
{
    public partial class Matchs
    {
        [Key]
        public int id { get; set; }

        public string numero_match { get; set; }
        public DateTime date_heure_match { get; set; }
        public int numero_journee { get; set; }
        public string categorie_equipe_local { get; set; }
        public string score_match { get; set; }
        public string equipe_local { get; set; }
        public string equipe_adverse { get; set; }
        public string lieu_match { get; set; }
        public string lien_feuille_de_match { get; set; }

    }
}
